src/js/components/Button/Button.js

```diff
@@ -1,22 +1,20 @@
 import React, { cloneElement, Children, Component } from 'react';
 import { compose } from 'recompose';
 
+import { colorIsDark, normalizeBackground, normalizeColor } from '../../utils';
 import { Box } from '../Box';
 import { withFocus, withForwardRef, withTheme } from '../hocs';
-import {
-  colorIsDark, colorForName, normalizeBackground, normalizeColor,
-} from '../../utils';
 
 import { StyledButton } from './StyledButton';
 
 const isDarkBackground = (props) => {
   const backgroundColor = normalizeBackground(normalizeColor(
     props.color || props.theme.button.primary.color
-    || props.theme.global.control.color || 'brand',
```

Comment:
this is just a reminder. we need to add this to our list of breaking changes in the PR description

Reply:
I've added a note at the bottom of the pull request details.